bundles/org.eclipse.ui.workbench/Eclipse UI/org/eclipse/ui/internal/dialogs/ViewsPreferencePage.java

```diff
@@ -114,6 +117,8 @@ public class ViewsPreferencePage extends PreferencePage implements IWorkbenchPre
 	private boolean highContrastMode;
 
 	private Button themingEnabled;
+	private Button updateOnRuntime;
```

Comment:
Can we call this `rescaleAtRuntime` to be consistent with the general naming? Same applies to other variables in this class.